Tests/RevenueCatUITests/Data/__Snapshots__/PaywallDataValidationTests/testValidateMissingPaywallWithSpanishLocalization-watchOS.1.json

```diff
@@ -11,7 +11,7 @@
       }
     },
     "display_restore_purchases" : true,
-    "images" : {
+    "images_heic" : {
       "background" : "background.jpg",
```

Comment:
That comes from this:
```swift
static let backgroundImage = "background.jpg"
```

The background image embedded in the framework is jpg. When creating a custom `PaywallData.Configuration` I chose to set `images_heic` instead of `images`, so it gets serialized like this.